### PR TITLE
default xcode spacing

### PR DESCRIPTION
**Pull request by sallyransom**
@iovortex @tLewisII @Cananito @yfujiki please review

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -120,9 +120,9 @@ let myClass = MyModule.UsefulClass()
 
 ## Spacing
 
-* Indent using 2 spaces rather than tabs to conserve space and help prevent line wrapping. Be sure to set this preference in Xcode as shown below:
+* Indent using Xcode's default 4 spaces (U.S. tabs). If your defaults do not use 4 spaces, be sure to set this preference in Xcode as shown below:
```
Comment (Cananito) [inline README.markdown]: Tabs change in certain countries?

**Comment by Cananito** [comment]
@sallyransom I'll +2 this: :+1: :+1:

**Comment by yfujiki** [comment]
+1. I am in 4 space camp as well. 

But on the other hand, I am guessing that the reason RW went for 2 spaces specifically for swift is for optional chaining. We can easily have deep indentation and it is sort of promoted in this language. (although it is alleviated in Swift 1.2)

```
if let ... {
    if let ... {
        if let ... {
                 ...
        }
    }
}
```

I am personally curious why different languages have different favored conventions (Ruby 2, Javascript 2, Java 4, C# 4 ...). I googled a bit but didn't get anything solid.

File: README.markdown (hunk 1 above)
Comment (yfujiki) [inline README.markdown]: Probably ASCII tab, is what is meant here?

**Comment by Cananito** [comment]
@yfujiki I favor 4 spaces because with pretty much all fonts you can easily distinguish all scopes. But on the `if let` pyramid of doom: ideally we shouldn't get into that situation through other fancy solutions.

**Comment by sallyransom** [comment]
i updated the comment by removing the part about the U.S. tabs. I've noticed in pairing with devs from other countries that their default tabs are sometime 2 spaces instead of 4 in Xcode so i just assumed it was a U.S. tab thing. 

Fortunately, we don't have to worry too much about the `if let` nest mess anymore since we can have multiple optional bindings separated by a comma now with Swift 1.2.

**Comment by Cananito** [comment]
@sallyransom I think it's ready for a freedom merge 🇺🇸

**Pull request merged**